<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="6" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-03-09T20:35:09 EST" name="Suite" finished-at="2021-03-09T20:35:09 EST" duration-ms="61">
    <groups>
      <group name="diningrooms">
        <method signature="FurnitureTest.Sofa()[pri:0, instance:Superstore.Walmart.FurnitureTest@3b938003]" name="Sofa" class="Superstore.Walmart.FurnitureTest"/>
        <method signature="FurnitureTest.Table()[pri:0, instance:Superstore.Walmart.FurnitureTest@3b938003]" name="Table" class="Superstore.Walmart.FurnitureTest"/>
        <method signature="ToysTest.car()[pri:0, instance:Superstore.Walmart.ToysTest@2b6856dd]" name="car" class="Superstore.Walmart.ToysTest"/>
      </group> <!-- diningrooms -->
    </groups>
    <test started-at="2021-03-09T20:35:09 EST" name="Test" finished-at="2021-03-09T20:35:09 EST" duration-ms="61">
      <class name="Superstore.Walmart.FurnitureTest">
        <test-method signature="Sofa()[pri:0, instance:Superstore.Walmart.FurnitureTest@3b938003]" started-at="2021-03-09T20:35:09 EST" name="Sofa" finished-at="2021-03-09T20:35:09 EST" duration-ms="6" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [b] but found [a]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [b] but found [a]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Superstore.Walmart.FurnitureTest.Sofa(FurnitureTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Sofa -->
        <test-method signature="Table()[pri:0, instance:Superstore.Walmart.FurnitureTest@3b938003]" started-at="2021-03-09T20:35:09 EST" name="Table" finished-at="2021-03-09T20:35:09 EST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Table -->
      </class> <!-- Superstore.Walmart.FurnitureTest -->
      <class name="Superstore.Walmart.ToysTest">
        <test-method signature="car()[pri:0, instance:Superstore.Walmart.ToysTest@2b6856dd]" started-at="2021-03-09T20:35:09 EST" name="car" finished-at="2021-03-09T20:35:09 EST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- car -->
      </class> <!-- Superstore.Walmart.ToysTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
